README.rst

```diff
@@ -301,3 +301,21 @@ token request:
         (default to: https://api.accounts.firefox.com/v1)
       - FXA_OAUTH_SERVER_URL: To select the oauth server url
         (default to: https://oauth.accounts.firefox.com/v1)
+
+
+
+=====================
+Contributing to PyFxA
+=====================
+
+The basic requirements are:
+- Python 3.12.2 or higher
+- Pip 24.0
+
+To get start: `python -m pip install -r dev-requirements.txt`
+
```

Comment:
Need to add:
- python setup.py build
- python setup.py install

Reply:
Read me updated.